kobo/apps/stripe/views.py

```diff
@@ -23,18 +23,31 @@ def get_queryset(self):
         return self.queryset.filter(
             livemode=settings.STRIPE_LIVE_MODE,
             customer__subscriber__users=self.request.user,
+        ).prefetch_related(
+            Prefetch(
+                'items',
+                queryset=SubscriptionItem.objects.select_related(
+                    'price__product'
+                ),
+            )
         )
 
 
 class ProductViewSet(viewsets.GenericViewSet, mixins.ListModelMixin):
+    """
+    Stripe Product + Prices
+
+    unit_amount is price in cents
+    """
+
```

Comment:
Yeah go for it - the "unit_amount is price in cents" was confusing so I documented it. Though maybe it can be documented on the field level help_text?

Reply:
I think leaving it  in the documentation might be better, since it's a read only API the help_text wouldn't appear anywhere other than the code